app/assets/stylesheets/application.css

```diff
@@ -333,3 +339,11 @@ input[type="submit"] {
   text-align: center;
   margin-top: 10px;
 }
+
+.back-button {
+  color: grey;
+  text-decoration: none;
+  font-size: 15px;
+  margin: 2.5px;
+  border: 1px solid grey;
+}
```

Comment:
Great job @Mutai-Gilbert 🤝

- [ ] Great job on adding this back button just like the previous reviewer suggested. However, kindly ensure this button can help in the navigation of your application. It is still very hard to navigate back to the home page.